apps/judicial-system/backend/src/app/formatters/rulingPdf.ts

```diff
@@ -215,7 +215,7 @@ export async function getRulingPdfAsString(
     .text('Málsatvik')
     .font('Helvetica')
     .fontSize(12)
-    .text(existingCase.caseFacts, {
+    .text(existingCase.courtCaseFacts, {
```

Comment:
This will cause a problem is someone retrieves the ruling pdf for an older case.

We have two ways to address this:

1.  use existingCase.courtCaseFacts || existingCase.caseFacts when generating the pdf
2. copy the value from case_facts to court_case_facts during the db migration

Reply:
Good catch 👀

I think way nr 2 is the way to go because caseFacts should never be under málsatvik anymore so `existingCase.courtCaseFacts || existingCase.caseFacts` is incorrect.

Reply:
Implemented in the migration script.